FirebaseAuth/Sources/AuthProvider/OAuth/FIROAuthProvider.m

```diff
@@ -272,19 +284,25 @@ - (void)getHeadFulLiteURLWithEventID:(NSString *)eventID
                                      }
                                      __strong __typeof__(self) strongSelf = weakSelf;
                                      NSString *bundleID = [NSBundle mainBundle].bundleIdentifier;
-                                     NSString *clienID = strongSelf->_auth.app.options.clientID;
+                                     NSString *clientID = strongSelf->_auth.app.options.clientID;
+                                     NSString *firebaseAppID =
```

Comment:
nit: using both firebaseAppID and googleAppID looks confusing. Maybe simply name the variable as `appID`?

Reply:
Done.